### DISCUSSION TITLE
How to setup Socket.io with Next.js

### DISCUSSION THREAD

**Discussion by arsekil** [socketio/socket.io#4865] [Q&A] [answered]
Hello fellow devs,

I find it hard to start working on my implementation of a chat functionality, b cause I didn't any useful tutorials or videos on how to srtup Socket.io with Next.js.

I have tried some similar implementation approaches, but none of them worked.

Does anyone have a detailed setup guide for this case, please?

**Comment by darrachequesne** [comment, marked as answer]
For future readers:

Please check our Next.js guide: https://socket.io/how-to/use-with-nextjs
